lib/graphql/anycable/cleaner.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+module GraphQL
+  module Anycable
+    module Cleaner
+      extend self
```

Comment:
This is `extend self`, as `module_function` is a visibility modifier and I cannot define `private` methods for it. I mean, I can, but they'll be instance only methods, and cannot be used from the singleton class invocation of `GraphQL::Anycable::Cleaner.clean` 😅